npm-dependency error when build docker container

### Product versie / Product version

1.28.0

### Omschrijf het probleem / Describe the bug

Taiga related issue Dimpact 484 Open Zaak: problem building docker image

reporter: Edgar Vonk

Good morning. We have a problem building a Docker images from the latest versions of (1.26, 1.27 en 1.28) of open-zaak from the repository https://github.com/open-zaak/open-zaak . This worked fine last week, but now it fails on a  NPM-dependency:


```
 ERROR in ./src/openzaak/sass/screen.scss
 Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
 HookWebpackError: Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
 Error: Cannot find module 'sass'
```

A number of versions in package.json is set to latest. I suspect something changed in a NPM-repository which causes the build to fail. We need this to build arm64-images of open-zaak. Can you please have a look at this?

### Stappen om te reproduceren / Steps to reproduce

_No response_

### Verwacht gedrag / Expected behavior

_No response_

--- a/CHANGELOG.rst
+++ b/CHANGELOG.rst
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+1.28.1 (2026-05-04)
+-------------------
+
+**Bugfixes**
+
+* Add missing sass dependency that caused failing Docker frontend builds
+
 1.28.0 (2026-04-14)
 -------------------
 

--- a/README.en.md
+++ b/README.en.md
@@ -6,7 +6,7 @@
 
 | | | |
 |-|-|-|
-| **Version:** | 1.28.0 |
+| **Version:** | 1.28.1 |
 | **Source:** | https://github.com/open-zaak/open-zaak |
 | **Keywords:** | zaken, zaakgericht werken, zaken-api, catalogi-api, besluiten-api, documenten-api |
 | **PythonVersion:** | 3.12 |

--- a/README.rst
+++ b/README.rst
@@ -7,7 +7,7 @@ Open Zaak
 
 .. _`Read this in English`: README.en.md
 
-:Version: 1.28.0
+:Version: 1.28.1
 :Source: https://github.com/open-zaak/open-zaak
 :Keywords: zaken, zaakgericht werken, zaken-api, catalogi-api, besluiten-api, documenten-api
 :PythonVersion: 3.12

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -114,6 +114,8 @@ linkcheck_ignore = [
     r"https://www.npmjs.com/.*",
     "https://github.com/maykinmedia/django-setup-configuration/blob/main/README.rst#environment-variable-substitution",
     r"https://opentelemetry\.io/docs/.*",  # times out frequently, even with 30s timeouts
+    r"https://samenorganiseren.slack.com",  # raises 403
+    r"https://join.slack.com",  # raises 403
 ]
 
 linkcheck_request_headers = {

--- a/docs/development/getting_started.rst
+++ b/docs/development/getting_started.rst
@@ -29,7 +29,7 @@ You need the following libraries and/or programs:
 
 .. _Python: https://www.python.org/
 .. _Virtualenv: https://virtualenv.pypa.io/en/stable/
-.. _Pip: https://packaging.python.org/tutorials/installing-packages/#ensure-pip-setuptools-and-wheel-are-up-to-date
+.. _Pip: https://packaging.python.org/tutorials/installing-packages/
 .. _PostgreSQL: https://www.postgresql.org
 .. _PostGIS-extension: https://postgis.net/
 .. _Node.js: http://nodejs.org/

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "openzaak",
-  "version": "1.28.0",
+  "version": "1.28.1",
   "description": "Open Zaak",
   "main": "src/openzaak/static/bundles/openzaak-js.js",
   "directories": {

--- a/publiccode.yaml
+++ b/publiccode.yaml
@@ -7,8 +7,8 @@ publiccodeYmlVersion: '0.2'
 name: Open Zaak
 url: 'https://github.com/open-zaak/open-zaak.git'
 softwareType: standalone/backend
-softwareVersion: 1.28.0
-releaseDate: '2026-04-14'
+softwareVersion: 1.28.1
+releaseDate: '2026-05-04'
 logo: 'https://raw.githubusercontent.com/open-zaak/open-zaak/main/docs/logo.svg'
 platforms:
   - web

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 requires-python = "~= 3.12.12"
-version = "1.28.0"
+version = "1.28.1"
 name = "open-zaak"
 
 # Bumpversion configuration
@@ -8,7 +8,7 @@ name = "open-zaak"
 [tool.bumpversion]
 commit = false
 tag = false
-current_version = "1.28.0"
+current_version = "1.28.1"
 pre_commit_hooks = [
     "npm i",  # ensure that package-lock.json is updated
 ]

--- a/src/openzaak/__init__.py
+++ b/src/openzaak/__init__.py
@@ -3,6 +3,6 @@
 from .celery import app as celery_app
 
 __all__ = ("celery_app",)
-__version__ = "1.28.0"
+__version__ = "1.28.1"
 __author__ = "Maykin Media"
 __homepage__ = "https://github.com/open-zaak/open-zaak"

